ecommerce/monitoring/grafana/provisioning/datasources/datasources.yml

```
@@ -0,0 +1,26 @@
+apiVersion: 1
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)